src/postamble.js

```diff
@@ -163,22 +163,21 @@ function run() {
 
 #if WASM_WORKERS
   if (ENVIRONMENT_IS_WASM_WORKER) {
+    initRuntime();
```

Comment:
Why does the order of promise resolution vs `initRuntime` need to be flipped now?

Reply:
This is not longer part of this PR